cfpb/cfgov-refresh
# Removing video player code from common.js

All pages which contain the video-player have been converted to Wagtail. This PR removes the video player from commons.js. It will result in the common.js bundle, which is on every page, being 5kb smaller. I have also removed the Ustream code from the bundle because it's no longer used. I'll remove the Ustream related code in a subsequent PR. 

## Changes

- Modified files to remove vidoe player.js from commons.js. 

## Testing

1. Run `gulp build`
2. Visit `http://localhost:8000/about-us/events/archive-past-events/public-event-about-debt-collection-washington-dc/` and ensure the video player works.
3. Visit `http://localhost:8000/about-us/the-bureau/` and ensure that the video player works.
4. Visit `http://localhost:8000/about-us/careers/application-process/` and ensure the video player works. 

## Checklist
